File: test/nodejs/simple-convert/simple-test.js
const ConvertorioClient = require('convertorio-sdk');
const path = require('path');

const API_KEY = 'cv_b13aff4fa6e8b71234e370105d8faf1b4150e7ddfc109df349dc6db4eae3';

async function runTest() {
    console.log('Iniciando conversión...');

    try {
        const client = new ConvertorioClient({
            apiKey: API_KEY,
            baseUrl: 'https://api.convertorio.com'
        });

        const result = await client.convertFile({
            inputPath: path.join(__dirname, 'test-image.png'),
            targetFormat: 'jpg',
            outputPath: path.join(__dirname, 'output.jpg')
        });

        console.log('Conversión finalizada');
        console.log(`Job ID: ${result.jobId}`);
        console.log(`Tiempo: ${result.processingTime}ms`);

    } catch (error) {
        console.error('Error:', error.message);
        process.exit(1);
    }
}

runTest();

File: test/python/simple-convert/simple-test.py
"""
Simple Convertorio SDK Test

Basic conversion test without advanced metadata options.
"""

from pathlib import Path
from convertorio_sdk import ConvertorioClient

API_KEY = 'cv_b13aff4fa6e8b71234e370105d8faf1b4150e7ddfc109df349dc6db4eae3'


def run_test():
    print('Iniciando conversión...')

    try:
        client = ConvertorioClient(
            api_key=API_KEY,
            base_url='https://api.convertorio.com'
        )

        current_dir = Path(__file__).parent
        result = client.convert_file(
            input_path=str(current_dir / 'test-image.png'),
            target_format='jpg',
            output_path=str(current_dir / 'output.jpg')
        )

        print('Conversión finalizada')
        print(f"Job ID: {result['job_id']}")
        print(f"Tiempo: {result['processing_time']}ms")

    except Exception as error:
        print(f'Error: {error}')
        exit(1)


if __name__ == '__main__':
    run_test()
